styles/_pages.scss

```diff
@@ -157,55 +157,245 @@
 }
 
 #use-case-card-grid {
-  @include media-breakpoint-up(lg) {
-    margin-top: -40px; // offset the 40px margin of the cols
-  }
+  // @include media-breakpoint-up(lg) {
+  //   margin-top: -40px; // offset the 40px margin of the cols
+  // }
 }
 
 #validator-graphic {
   content: url(../img/validators.svg);
 }
 
 .page-uses {
+  
+  .container-new {
+    padding-left: 16px;
+    padding-right: 16px;
+  }
+
+  h1 {
+    font-size: 42px;
+  }
+
+
   &::before {
+    transform: scaleX(-1);
     background-image: url(../img/backgrounds/use-cases-blue.svg);
   }
-  @each $usecase in "micropayments", "wallets", "exchanges", "stablecoins", "nft", "defi", "cbdc" {
-    ##{$usecase} {
-      content: url("../img/icons/#{$usecase}.svg");
-    }
+
+  .card-grid {
+    grid-gap: 8px
+  }
+
+  .card-grid img {
+    max-height: 40px;
+  }
+
+  /* Cleanup bootstrap modal */
+  .modal {
+    padding: 0; 
+  }
+  .modal-content {
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    pointer-events: auto;
+    background-color: #000;
+
+    background-clip: padding-box; 
+    border: none;
+    border-radius: 0;
+    box-shadow: none;
+    outline: none;
+
+    background: #111112;
+  }
+
+  .modal-header {
+    border: none;
+    background: #111112;
+    box-shadow: 0px 1px 2px #000000;
+  }
+  .modal-header .cancel .chevron {
+    transform: rotate(90deg);
+  }
+  .modal-header .apply .chevron {
+    transform: rotate(-90deg);
+  }
+
+  .modal-footer {
+    border: none;
+    background: #111112;
+    box-shadow: 0px -1px 2px #000000;
+    
+    align-items: unset; 
+    padding: 0.75rem;
+    flex-direction: column;
+    flex-wrap: wrap;
+  }    
+
+
+// 
```

Comment:
Was there supposed to be text to this comment?

Reply:
removed